File: testcases/popping_blocks/popping_blocks_checkfile.js


while(line=readline()){
    line = line.substring(1, line.length - 1);
    var strArr = line.split(",")
    var i;
    var arr = [];
    for(i=0; i<strArr.length; i++){
        arr.push(strArr[i].substring(1, strArr[i].length - 1));
    }
    print(JSON.stringify(popping_blocks(arr)));

}

File: testcases/popping_blocks/popping_blocks_checkfile.java




    public static void main(String[] args) throws Exception {
        // Your code here!
        Scanner sc = new Scanner(System.in);

        // String input
        while (sc.hasNextLine()){
            String str = sc.nextLine();
            if (str != null && str.length() > 0) {
                str = str.substring(0, str.length() - 1);
                str = str.substring(1, str.length());
            }
            String[] strArray = str.split(",");
            String[] res = this.popping_blocks(strArray);
            System.out.println(Arrays.toString(res));
        }

    }
}